docs/docs/spark-procedures.md

```diff
@@ -972,4 +972,91 @@ CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id =>
 Collect statistics of the snapshot with id `snap1` of table `my_table` for columns `col1` and `col2`
 ```sql
 CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id => 'snap1', columns => array('col1', 'col2'));
-```
\ No newline at end of file
+```
+
+## Table Replication
+
+The `rewrite-table-path` assists in moving or copying an Iceberg table from one location to another.
+
+### `rewrite-table-path`
+
+This procedure writes a new copy of the Iceberg table's metadata files where every path has had its prefix replaced.
+The newly rewritten metadata files enable moving or coping an Iceberg table to a new location.
+After copying both metadata and data to the desired location, the replicated iceberg
+table will appear identical to the source table, including snapshot history, schema and partition specs.
+
+!!! info
+    This procedure only creates metadata for an existing Iceberg table modified for a new location. The produced file_list can be used for copying rewritten metadata and data files to the new location.
```

Comment:
How about:

``` This procedure only prepares metadata for an existing Iceberg table in preparation for a copy or move to a new location.  The copying or moving of files is outside its scope. ```

Maybe the file_list could be left out as its explained in the output below?

Reply:
Sounds good. Updated but I think I think keep the existing tone of not part of procedure reads a bit better than outside its scope, let me know if you feel otherwise